Sources/Overload/OvEditor/src/OvEditor/Core/EditorActions.cpp

```diff
@@ -483,9 +483,10 @@ OvMaths::FVector3 OvEditor::Core::EditorActions::CalculateActorSpawnPoint(float
 	return sceneView.GetCameraPosition() + sceneView.GetCameraRotation() * OvMaths::FVector3::Forward * p_distanceToCamera;
 }
 
-OvCore::ECS::Actor & OvEditor::Core::EditorActions::CreateEmptyActor(bool p_focusOnCreation, OvCore::ECS::Actor* p_parent)
+OvCore::ECS::Actor & OvEditor::Core::EditorActions::CreateEmptyActor(bool p_focusOnCreation, OvCore::ECS::Actor* p_parent, const std::string& p_name)
```

Comment:
`bool p_focusOnCreation` can be set const

Reply:
Out of the scope of this PR.
Do note that I'm not a fan of "consting" everything outside of pointer and references. `p_focusOnCreation` here is already copied, and is local to the function.
Anyway, out of the scope